src/quacc/recipes/emt/elastic.py

```diff
@@ -66,11 +69,21 @@ def bulk_to_deformations_flow(
         [relax_job, static_job],
         param_swaps=job_params,
         decorators=job_decorators,
-    )
+    )  # type: ignore
+
+    if pre_relax:
+        if run_static:
+            undeformed_relax = relax_job_(atoms, relax_cell=True)
+            undeformed_result = static_job_(undeformed_relax["atoms"])
+        else:
+            undeformed_result = relax_job_(atoms, relax_cell=True)
```

Comment:
We should run `undeformed_result = relax_job_(atoms, relax_cell=True)` first regardless. And then `if run_static` we can run the static. We have some code duplication here otherwise. See below.